examples/api/test/widgets/widget_state/widget_state_mouse_cursor.0_test.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_api_samples/widgets/widget_state/widget_state_mouse_cursor.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Displays ListTile', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const example.WidgetStateMouseCursorExampleApp(),
+      );
+
+      expect(find.byType(ListTile), findsOneWidget);
+      expect(find.text('ListTile'), findsOneWidget);
+    },
+  );
```

Comment:
These tests LGTM, aside from a small whitespace nit.

Rather than

```dart
testWidgets('Displays ListTile', (WidgetTester tester) async {
    // ...
  },
);
```

I would prefer:

```dart
testWidgets('Displays ListTile', (WidgetTester tester) async {
  // ...
});
```